go.mod

```diff
@@ -13,7 +13,6 @@ require (
 	github.com/containerd/containerd v1.7.12
 	github.com/cyphar/filepath-securejoin v0.2.4
 	github.com/distribution/distribution/v3 v3.0.0-20221208165359-362910506bc2
-	github.com/evanphx/json-patch v5.7.0+incompatible
```

Comment:
Why using gopkg.in URL instead of `github.com/evanphx/json-patch/v5` ?

Does it make any difference ?

Reply:
I’m using the gopkg.in URL because that’s what’s documented... @evanphx would you care to weigh in?

Reply:
I forgot all about that part. I should remove it, use the direct GitHub url.

Reply:
I’ve updated this PR to do that and opened https://github.com/evanphx/json-patch/pull/203 to fix the documentation.

Reply:
@skitt Thank you!